editor/store/selectors.js

```diff
@@ -354,12 +353,7 @@ export function getEditedPostExcerpt( state ) {
  * @return {string} Preview URL.
  */
 export function getEditedPostPreviewLink( state ) {
-	const link = state.currentPost.link;
-	if ( ! link ) {
-		return null;
-	}
-
-	return addQueryArgs( link, { preview: 'true' } );
+	return getCurrentPost( state ).preview_link || null;
```

Comment:
I think it would be better to use `getEditedPostAttribute( 'preview_link' )` so we could get rid of `getCurrentPost` at some point.

Reply:
Can you make an issue for this?

Reply:
Doing PR

Reply:
#6894